test/math/SafeMath.test.js

```diff
@@ -1,60 +1,91 @@
-import assertRevert from '../helpers/assertRevert';
-const assertJump = require('../helpers/assertJump');
-var SafeMathMock = artifacts.require('SafeMathMock');
+import assertJump from '../helpers/assertJump';
+const BigNumber = web3.BigNumber;
+const SafeMathMock = artifacts.require('SafeMathMock');
 
-contract('SafeMath', function (accounts) {
-  let safeMath;
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('SafeMath', () => {
+  const MAX_UINT = new BigNumber('115792089237316195423570985008687907853269984665640564039457584007913129639935');
```

Comment:
Do we not want to do `new BigNumber(1).pow(256).sub(1)` or whatever that was to get the max number?